internal/cfgplugins/qos.go

```diff
@@ -260,3 +261,90 @@ func enableQosRemarkDscpCliConfig(dut *ondatra.DUTDevice, classifierName string,
+}
+
+func configureTwoRateThreeColorSchedulerFromOC(qos *oc.Qos, schedulerName, policerName string, cirValue, pirValue uint64, burstSize, sequenceNumber uint32, queueName string) {
+	sp := qos.GetOrCreateSchedulerPolicy(schedulerName)
+	sp.Name = ygot.String(schedulerName)
+	sched := sp.GetOrCreateScheduler(sequenceNumber)
+	sched.Sequence = ygot.Uint32(1)
+	sched.Type = oc.QosTypes_QOS_SCHEDULER_TYPE_TWO_RATE_THREE_COLOR
+	input := sched.GetOrCreateInput(policerName)
+	input.InputType = oc.Input_InputType_QUEUE
+	input.Queue = ygot.String(queueName)
+	trtc := sched.GetOrCreateTwoRateThreeColor()
+	trtc.Cir = ygot.Uint64(cirValue)
+	trtc.Pir = ygot.Uint64(pirValue)
+	trtc.Bc = ygot.Uint32(burstSize)
+	trtc.Be = ygot.Uint32(burstSize)
+	trtc.GetOrCreateExceedAction().Drop = ygot.Bool(false)
+	trtc.GetOrCreateViolateAction().Drop = ygot.Bool(true)
+}
+
+func configureTwoRateThreeColorSchedulerFromCLI(t *testing.T, dut *ondatra.DUTDevice, schedulerName, className string, cirValue, pirValue uint64, burstSize, queueId uint32) {
+	cliConfig := fmt.Sprintf(`
+policy-map type quality-of-service %s
+   class %s
+   set traffic-class %d 
+   police rate %d bps burst-size %d bytes rate %d bps burst-size %d bytes
+!
+`, schedulerName, className, queueId, cirValue, burstSize, pirValue, burstSize)
+
+	helpers.GnmiCLIConfig(t, dut, cliConfig)
+}
+
+func NewTwoRateThreeColorScheduler(t *testing.T, dut *ondatra.DUTDevice, qos *oc.Qos, schedulerName, policerName, className string, cirValue, pirValue uint64, burstSize, sequenceNumber, queueId uint32, queueName string) {
+	if deviations.QosTwoRateThreeColorPolicerOCUnsupported(dut) {
+		configureTwoRateThreeColorSchedulerFromCLI(t, dut, schedulerName, className, cirValue, pirValue, burstSize, queueId)
+	} else {
+		configureTwoRateThreeColorSchedulerFromOC(qos, schedulerName, policerName, cirValue, pirValue, burstSize, sequenceNumber, queueName)
+	}
+}
+
+func ApplyQosPolicyOnInterface(t *testing.T, dut *ondatra.DUTDevice, qos *oc.Qos, interfaceName, policyName string) {
+	if deviations.QosSchedulerIngressPolicer(dut) {
+		applyQosPolicyOnInterfaceFromCLI(t, dut, interfaceName, policyName)
+	} else {
+		applyQosPolicyOnInterfaceFromOC(qos, interfaceName, policyName)
+	}
+}
+
+func applyQosPolicyOnInterfaceFromOC(qos *oc.Qos, intfName, policyName string) {
+	intf := qos.GetOrCreateInterface(intfName)
+	intf.InterfaceId = ygot.String(intfName)
+	in := intf.GetOrCreateInput()
+	in.GetOrCreateSchedulerPolicy().Name = ygot.String(policyName)
+}
+
+func applyQosPolicyOnInterfaceFromCLI(t *testing.T, dut *ondatra.DUTDevice, intfName, policyName string) {
+	cliConfig := fmt.Sprintf(`
+interface %s
+   service-policy type qos input %s
+!
+`, intfName, policyName)
+	helpers.GnmiCLIConfig(t, dut, cliConfig)
+}
+
+func GetPolicyCLICounters(t *testing.T, dut *ondatra.DUTDevice, policyName string) string {
+	cliConfig := fmt.Sprintf("show policy-map type qos %s counters", policyName)
+	return runCliCommand(t, dut, cliConfig)
```

Comment:
why use a different style here?  Probably stick to the same style, using 
	helpers.GnmiCLIConfig(t, dut, cliConfig)